com.civicrm.hrjobroles/js/src/job-roles/controllers/hr-job-roles-controller.js

```diff
@@ -126,10 +126,10 @@ define([
              * @param end
              * @param {function} error
```

Comment:
If you change a function/method signature, please also update its documentation to reflect that